blocks/bricks/__init__.py

```diff
@@ -658,10 +658,17 @@ def add_property(f):
         return add_property
 
     def __getattr__(self, attr):
+        # Prevents infinite recursion when no attribute "f" is available
+        # (happens during serialization under unknown circumstances)
+        try:
+            self_f = self.__getattribute__("f")
```

Comment:
I don't think this makes much sense; `__getattr__` is only called if the attribute (`attr`) couldn't be found as a usual attribute, so calling `__getattribute__` again will always fail. If you just use
```python
if attr == 'f':
    return {}
```
and not change anything else, does that work?